Cracking Fenwick Tree to the root V1/Solutions/cmpeguerog/D - Points in Rectangle/Points in Rectangle.cpp

```diff
@@ -0,0 +1,92 @@
+#include <bits/stdc++.h>
```

Comment:
It would be nice if everybody use the same convention, e.g. problem  folder should match [A-Z]{1,2} or something similar